tests/baselines/reference/constEnum2.errors.txt

```diff
@@ -1,10 +1,9 @@
 tests/cases/conformance/constEnums/constEnum2.ts(10,9): error TS2474: const enum member initializers can only contain literal values and other computed enum values.
 tests/cases/conformance/constEnums/constEnum2.ts(11,9): error TS2474: const enum member initializers can only contain literal values and other computed enum values.
 tests/cases/conformance/constEnums/constEnum2.ts(12,5): error TS1357: An enum member name must be followed by a ',', '=', or '}'.
-tests/cases/conformance/constEnums/constEnum2.ts(12,9): error TS2474: const enum member initializers can only contain literal values and other computed enum values.
 
 
-==== tests/cases/conformance/constEnums/constEnum2.ts (4 errors) ====
+==== tests/cases/conformance/constEnums/constEnum2.ts (3 errors) ====
     // An enum declaration that specifies a const modifier is a constant enum declaration.
     // In a constant enum declaration, all members must have constant values and
     // it is an error for a member declaration to specify an expression that isn't classified as a constant enum expression.

```

Comment:
Actually, comments are fine. What's changed here is that `g = CONST` is now an allowed declaration because we've added the ability to compile-time evaluate references to `const` declarations.

That said, the test is missing a comma after one of the declarations, which is rather pointless. I will fix that.